torch/testing/_internal/common_cuda.py

```diff
@@ -2,14 +2,14 @@
 
 import torch
 import torch.cuda
-from torch.testing._internal.common_utils import TEST_WITH_ROCM, TEST_NUMBA
+from torch.testing._internal.common_utils import TEST_NUMBA
 
 
 TEST_CUDA = torch.cuda.is_available()
 TEST_MULTIGPU = TEST_CUDA and torch.cuda.device_count() >= 2
 CUDA_DEVICE = TEST_CUDA and torch.device("cuda:0")
 # note: if ROCm is targeted, TEST_CUDNN is code for TEST_MIOPEN
-TEST_CUDNN = TEST_CUDA and (TEST_WITH_ROCM or torch.backends.cudnn.is_acceptable(torch.tensor(1., device=CUDA_DEVICE)))
+TEST_CUDNN = TEST_CUDA and torch.backends.cudnn.is_acceptable(torch.tensor(1., device=CUDA_DEVICE))
```

Comment:
dumb question: why we no longer need TEST_WITH_ROCM here?

Reply:
The `is_acceptable` function would previously return `False` for ROCm builds, so we would override it by using `TEST_WITH_ROCM`, but with my changes, the function should return `True` even for ROCm builds, rendering the workaround unnecessary.